### ISSUE TITLE
Cannot log out user

### ISSUE THREAD

**Issue by teocomi** [psignoret/aad-sso-wordpress#119]
Hello, first of all thanks for the great plugin, I've installed it and it works very well a part from the logging out part. Here the steps to reproduce it:

- user A logs in
- user A logs out (from wordpress and from AAD)
- user B logs in from the same machine&browser
- he appears logged in as user A

Seems like the plugin is not deleting the token cache? Or might it be some misconfiguration on my side?
Thanks

**Comment by bradkovach** [comment]
Could you post a list or screenshot of active plugins?  Also, let us know if you're using the most recent version of the plugin?

**Comment by teocomi** [comment]
Sure, it's a new wordpress installation and the latest version of the pugin (respectively 4.7.2 and 0.6a)

![image](https://cloud.githubusercontent.com/assets/2679513/23121089/ff6fb298-f756-11e6-8678-5e83343948de.png)

The theme is http://www.wpexplorer.com/pronto-wordpress-theme/

I forgot to mention that the only way I managed to log out the other user was by cleaning the browser cache.

**Comment by psignoret** [comment]
How are you logging out the user (from the blog and from Azure AD)?

**Comment by teocomi** [comment]
Yes: I click on "Log Out" in the blog and get redirected to the wordpress login page, then I click "Sign Out" and it goes through my company's azure logout correctly.
When I try to log in again, and select to use another user account, I then appear logged in wordpress as the first user...

![image](https://cloud.githubusercontent.com/assets/2679513/23133909/b4647174-f78a-11e6-87e6-37ec4f55d2e9.png)

**Comment by rdroshan24** [comment]
I am facing the same issue as @teocomi have posted.
After logout, as @teocomi mentioned, when I try to log in again, and select to use another user account, it directly take me to the application without asking for password.
I can not get logout completely unless and until I close the browser.
FYI, I have used wp_logout_url(home_url()) function to logout from Portal.
